src/github/dto/create.org.dto.ts

```diff
@@ -0,0 +1,82 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsString,
+  IsEmail,
+  IsEnum,
+  IsArray,
+  IsBoolean,
+  IsOptional,
+  Length,
+} from 'class-validator';
+import { OrganizationStatus } from 'src/common/constants';
+
+export class CreateOrgDto {
+  @ApiProperty({
+    description: 'The host of the organization (e.g., GitHub)',
+    example: 'github',
+    default: 'github',
+  })
+  @IsString()
+  @Length(1, 255)
+  host: string;
+
+  @ApiProperty({
+    description: 'The unique name of the organization',
+    example: 'example-org',
+  })
+  @IsString()
+  @Length(1, 255)
+  organizationName: string;
```

Comment:
PullProofAI

The property `organizationName` should follow a consistent naming convention. Consider renaming it to `name`.

```diff
-   organizationName: string;
+   name: string;
```

#### Issue: Inconsistent naming convention for `adminUserName`


<!-- This is an auto-generated comment by PullProofAI -->